src/overlay/Peer.cpp

```diff
@@ -389,10 +389,15 @@ Peer::sendAuth()
     ZoneScoped;
     StellarMessage msg;
     msg.type(AUTH);
-    if (mAppConnector.getConfig().ENABLE_FLOW_CONTROL_BYTES)
+#ifdef BUILD_TESTS
+    if (!mAppConnector.getOverlayManager()
+             .isFlowControlBytesDisabledForTesting())
     {
         msg.auth().flags = AUTH_MSG_FLAG_FLOW_CONTROL_BYTES_REQUESTED;
     }
+#else
```

Comment:
Nit: This `#else` isn't necessary, you can just wrap the if expression:
```
#ifdef BUILD_TESTS
    if (!mAppConnector.getOverlayManager()
             .isFlowControlBytesDisabledForTesting())
#endif
```

Reply:
Done!